From dfa7386321aee36006dcf014b825ec99679a35d9 Mon Sep 17 00:00:00 2001
From: Masahiro Yamada <masahiroy@kernel.org>
Date: Mon, 26 Sep 2022 18:02:25 +0900
Subject: [PATCH 28/39] mksysmap: update comment about __crc_*

Since commit 7b4537199a4a ("kbuild: link symbol CRCs at final link,
removing CONFIG_MODULE_REL_CRCS"), __crc_* symbols never become
absolute.

Keep ignoring __crc_*, but update the comment.

Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
---
 scripts/mksysmap | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/scripts/mksysmap b/scripts/mksysmap
index ad8bbc522..bc5396e25 100755
--- a/scripts/mksysmap
+++ b/scripts/mksysmap
@@ -37,8 +37,8 @@
 
 # readprofile starts reading symbols when _stext is found, and
 # continue until it finds a symbol which is not either of 'T', 't',
-# 'W' or 'w'. __crc_ are 'A' and placed in the middle
-# so we just ignore them to let readprofile continue to work.
-# (At least sparc64 has __crc_ in the middle).
-
+# 'W' or 'w'.
+#
+# Ignored prefixes:
+#  __crc_               - modversions
 $NM -n $1 | grep -v '\( [aNUw] \)\|\(__crc_\)\|\( \$[adt]\)\|\( \.L\)\|\( L0\)' > $2
-- 
2.38.0.rc1.6.g4fd6c5e444

